checkov/terraform/graph_builder/local_graph.py

```diff
@@ -495,13 +517,14 @@ def update_vertices_configs(self) -> None:
             changed_attributes = filter_sub_keys(changed_attributes)
             self.update_vertex_config(vertex, changed_attributes)
 
-    def update_vertex_config(self, vertex: TerraformBlock, changed_attributes: Union[List[str], Dict[str, Any]], dynamic_blocks: bool = False) -> None:
+    @staticmethod
+    def update_vertex_config(vertex: TerraformBlock, changed_attributes: Union[List[str], Dict[str, Any]], dynamic_blocks: bool = False) -> None:
         if not changed_attributes:
             # skip, if there is no change
             return
 
         vertex_name = vertex.name
-        updated_config = pickle_deepcopy(vertex.config)
+        updated_config: Any = pickle_deepcopy(vertex.config)
```

Comment:
maybe `dict[str, Any]` is better here

Reply:
the problem is a few lines later it overridden with a different type. But I can try to make to separate it somehow to be a bit more precise on the type.